components-2/readme.md

```diff
@@ -0,0 +1,166 @@
+
+- [x] [HTML-CSS](https://curriculum.codeyourfuture.io/html-css/)
+- [x] [JS1](https://curriculum.codeyourfuture.io/js1/)
+- [x] [JS2](https://curriculum.codeyourfuture.io/js2/)
+
+And in particular, these workshops:
+
+- [x] [Git](https://curriculum.codeyourfuture.io/induction/sprints/1/day-plan/#git-workshop)
+- [x] [Devtools](https://curriculum.codeyourfuture.io/html-css/sprints/1/day-plan/#devtools-workshop)
+- [x] [Components 101](https://curriculum.codeyourfuture.io/html-css/sprints/4/day-plan/#components-workshop)
+
+If you missed these workshops in your cohort, it's a good idea to try to do them in the week (online) first. Record your session for people that can't attend. Everyone should try to do the workshops first, either in groups or solo if necessary.
+
+The overall mission is to compose "components" / groupings of HTML elements to build a page in [together.html](https://cyf-workshops.netlify.app/components-2/together.html).
+
+But this time, we're going to use JavaScript functions to build our components.
+
+## Learning Objectives
+
+```objectives
+- [ ] 🤩 Work on a complex sub-branch workflow
+- [ ] 🚢 Import a function from a module
+- [ ] 🪄 Create a component on a web page
+- [ ] 📖 Write a function to build a component from a documented pattern
+- [ ] 👪🏿 Collaborate to build a page
+```
+
+<!--{{<note type="activity" title="Set up your working branch 15m">}}-->
+
+### Set up your working branch
+
+Everyone should do this so everyone is set up correctly.
+
+1. You will be working on a special branch called [FOR-USE-IN-CLASS](https://github.com/CodeYourFuture/CYF-Workshops/tree/FOR-USE-IN-CLASS).
+1. First, **clone** [the workshops repository](https://github.com/CodeYourFuture/CYF-Workshops/)
+1. Checkout `git checkout FOR-USE-IN-CLASS`
+1. Your cohort has a named _sub branch_. Everyone, check that out next with `git checkout COHORT` (eg `NW6`). This is your working branch:
+
+```mermaid
+gitGraph
+    commit "main"
+    branch FOR-USE-IN-CLASS
+    checkout FOR-USE-IN-CLASS
+    branch NW6
+    checkout NW6
+    branch feature/header
+    commit
+    commit
+    checkout NW6
+    branch feature/footer
+    commit
+    commit
+    commit
+    checkout NW6
+    branch feature/card
+    commit
+    commit
+    commit
+    checkout NW6
+    branch feature/menu
+    commit
+    commit
+    commit
+    checkout NW6
+    merge feature/header
+    merge feature/footer
+    merge feature/card
+    merge feature/menu
+```
+
+Check you are in the right place with GitLens!
+
+<!--{{</note>}}-->
+
+<!--{{<note type="tip" title="Your working branch is your cohort name">}}-->
+
+Your COHORT branch is your cohort name. It might be NW6, WM5, GL7...You're working in your COHORT branch and NOT in main.
+
+<!--{{</note>}}-->
+
+Next, divide the class into four teams. Use the [team generator](teams.html) to help you do this.
```

Comment:
```suggestion
Next, divide the class into four teams: header, footer, card, menu. Use the [team generator](teams.html) to help you do this.
```

Is it worth noting that this link seems to be broken in the components-1 workshop when on the curriculum site?